src/testlink.ts

```diff
@@ -72,7 +72,10 @@ export default class TestLink {
      * Establish connection
      */
     connect(): void {
-        this.rpcClient = xmlrpc.createClient({host: this.host, port: this.port, path: this.rpcPath});
+        let options = { host: this.host, port: this.port, path: this.rpcPath };
```

Comment:
As `options` is unmutable here, `const` is a better choice.
```suggestion
        const options = { host: this.host, port: this.port, path: this.rpcPath };
```

Reply:
sure, that's true